src/components/Modals/Modal.vue

```diff
@@ -127,11 +127,6 @@ export default Vue.extend({
           this.rightButton = 'ADD';
           this.$refs.modalBodyComponent.next();
         } else {
-          const dropdown = document.getElementById(`dropdown-${this.semesterID}`);
-          const title = dropdown.value;
-
-          const key = title.substring(0, title.indexOf(':'));
-          const selectedReqs = this.$refs.modalBodyComponent.getSelectedReqs();
```

Comment:
These values are unused and I checked `.getSelectedReqs()` is simply a getter method without side effects, so I removed them.